tests/test_views.py

```diff
@@ -244,30 +244,33 @@ def test_multiple_pending(self):
         self.assertEqual(got, want)
 
     def test_closed(self):
-        """Closed renewals are not renewable."""
+        """Closed renewals are not returned to the view."""
         advantage = make_advantage()
         now = datetime.now(timezone.utc)
-        start = (now - timedelta(days=1)).isoformat()
-        end = (now + timedelta(days=1)).isoformat()
+        start = (now + timedelta(days=1)).isoformat()
         contract_info = {
             "renewals": [
                 {
                     "id": "1",
                     "status": "closed",
                     "actionable": True,
                     "start": start,
-                    "end": end,
+                },
+                {
```

Comment:
Let's add a renewal in error state here to check that this is not taken?

Reply:
Will be addressed by https://github.com/canonical-web-and-design/ubuntu.com/issues/8755